sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs

```diff
@@ -24,6 +26,7 @@ public partial class CreateRsaKeyOptions : Azure.Security.KeyVault.Keys.CreateKe
         public int? KeySize { get { throw null; } set { } }
         public Azure.Security.KeyVault.Keys.KeyType KeyType { get { throw null; } }
         public string Name { get { throw null; } }
+        public int? PublicExponent { get { throw null; } set { } }
```

Comment:
Is public exponent only valid for RSA keys?

Reply:
Yes. The swagger models get reused a lot and shouldn't always, but it's understandably easier. If they ever add support for other algorithms that require a public exponent, we can add it to their specialized options classes.